platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineManager.java

```diff
@@ -114,6 +116,7 @@ public void run() {
     }).start();
   }
 
+  @UiThread
```

Comment:
@tobrun Why are we requiring `@UiThread` for methods here? I'm thinking of issues if the developer wants to download regions from a background thread, say from a `Service`.